.eslintrc

```diff
@@ -43,7 +43,7 @@
     "prefer-arrow-callback": 1,
     "no-useless-concat": 1,
     "prefer-template": 1,
-    "no-console": 1,
```

Comment:
Why did you change this?

Reply:
I was using console methods that why i had to turn off no-console. I'll change it back to 1. :)